services/web/client/source/class/osparc/desktop/StudyEditor.js

```diff
@@ -474,6 +474,24 @@ qx.Class.define("osparc.desktop.StudyEditor", {
         }
       }, this);
 
+      // callback for node updates
+      const slotName3 = "nodeUpdated";
+      socket.removeSlot(slotName3);
```

Comment:
cool... but what is that ``removeSlot`` for?

Reply:
It remove the listener if it already exists. Without it, if you start the same pipeline twice, you also subscribe to that message two times.